app.js

```diff
@@ -1,20 +1,27 @@
 const express = require("express");
 const data = require('./db/data/test-data/index.js')
-const {getAllTopics, getAllEndpoints} = require('./Controllers/controller.js')
+const { getAllTopics, getAllEndpoints, getArticleById } = require('./Controllers/controller.js')
+const { handleCustomErrors, handleSQLErrors } = require('./Error handling/error-handling.js');
 
 const app = express()
 
 
 
 app.get('/api/topics', getAllTopics)
 app.get('/api', getAllEndpoints)
+app.get('/api/articles/:article_id', getArticleById)
 
+
+
+//Error handling
+app.use(handleCustomErrors)
+app.use(handleSQLErrors)
 app.all('/*', (req, res, next) => {
-    res.status(400).send({msg: 'Bad request'})
+    res.status(400).send({ msg: 'Bad request' })
```

Comment:
(This is technically for the previous ticket but) ,a non-existant endpoint should be a 404 rather than a 400 bad request 🕵️